routes/node.js

```diff
@@ -41,123 +49,117 @@ router.get("/", (req, res) => {
  * GET request which takes a nodeID and returns its data.
  */
 router.get("/get", (req, res) => {
-	const {mapID, nodeID} = req.body;
+	const { mapID, nodeID } = req.body;
 
 	if (!nodeID || !mapID) {
 		res.status(400).send("nodeID and mapID must be specified!");
 		return;
 	}
 
-	client.connect(err => {
+	client.connect((err) => {
 		if (err) {
 			throw err;
 		}
 
 		const db = client.db("blurp");
 		const collection = db.collection("maps");
 
-		collection.aggregate([
-			{$match: {mapID: mapID}},
-			{ $unwind: "$nodes"},
-			{ $match: {'nodes.nodeID': nodeID}},
-			{ $project: {nodes: 1, _id:0}}
-		])
-		.toArray()
-		.then(output => {
-			if ( output.length <= 0 ) {
-				res.status(400).send("Node not found")
-			}
-			else {
-				output = output[0].nodes;
-				res.status(200).json(output);
-			}
-		});
+		collection
+			.aggregate([{ $match: { mapID: mapID } }, { $unwind: "$nodes" }, { $match: { "nodes.nodeID": nodeID } }, { $project: { nodes: 1, _id: 0 } }])
+			.toArray()
+			.then((output) => {
+				if (output.length <= 0) {
+					res.status(400).send("Node not found");
+				} else {
+					output = output[0].nodes;
+					res.status(200).json(output);
+				}
+			});
 	});
 });
 
 /**
  * POST request which takes userID, mapID, name, and color, and creates a new node record.
  */
 router.post("/create", (req, res) => {
-	const {userID, mapID, nodeinfo} = req.body;
+	const { userID, mapID, nodeinfo } = req.body;
+	console.log(userID);
```

Comment:
Are these console logs supposed to be present in production?

Reply:
Whoops, I missed removing those. I just made a commit deleting those from this branch. Thanks for catching that!